src/asm_files.cpp

```diff
@@ -25,13 +25,12 @@ static vector<T> vector_of(ELFIO::section* sec) {
     return {(T*)data, (T*)(data + size)};
 }
 
-int create_map_crab(uint32_t map_type, uint32_t key_size, uint32_t value_size, uint32_t max_entries, ebpf_verifier_options_t options) {
-    // BPF_MAP_TYPE_ARRAY_OF_MAPS and
-    // BPF_MAP_TYPE_HASH_OF_MAPS are not yet supported.
-    if (map_type == 12 || map_type == 13) {
+int create_map_crab(EbpfMapType& map_type, uint32_t key_size, uint32_t value_size, uint32_t max_entries, ebpf_verifier_options_t options) {
```

Comment:
Why is this non-const? Do you expect it to be mutated at some point?

Reply:
It's because the radix_tree implementation doesn't have the appropriate methods marked const.  I believe we can add the consts back in the future once this is fixed but that requires a PR upstream first.

Reply:
Updated to be const